src/odemis/acq/stream/_static.py

```diff
@@ -1158,6 +1159,536 @@ def onSpectrumBandwidth(self, value):
 # we could simplify/merge the two stream classes.
 
 
+class TemporalSpectrumStream(StaticStream):
```

Comment:
Please name the stream StaticTemporalSpectrumStream to keep it the same as the other static streams. Also the later in stream/__init__.py there will be a class TemporalSpectrumStream, where the Settings-, Static-, and MDStream are registered.